components/autosize_textarea.tsx

```diff
@@ -17,42 +16,53 @@ type Props = {
 
 export default class AutosizeTextarea extends React.PureComponent<Props> {
     private height: number;
+    private textareaRef: React.RefObject<HTMLTextAreaElement>;
+    private reference: React.RefObject<HTMLTextAreaElement>;
     constructor(props: Props) {
         super(props);
 
         this.height = 0;
+
+        this.textareaRef = React.createRef();
+        this.reference = React.createRef();
```

Comment:
nit: can we keep the convention of naming this `this.referenceRef`

Reply:
Since the instance varibale name is already reference thought of not adding it initially.  Will fix this